src/pages/manageToken.tsx

```diff
@@ -87,6 +91,52 @@ export default function manageToken() {
                 </div>
             </div>
             </div>
+
+            <div className="modal fade" id="BurnTokens" aria-labelledby="exampleModalLabel" aria-hidden="true">
+            <div className="modal-dialog">
+                <div className="modal-content">
+                <div className="modal-header">
+                    <h5 className="modal-title" id="exampleModalLabel">Burn Tokens</h5>
+                    <button type="button" className="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                </div>
+                <div className="modal-body">
+                    <div className="row">
+                        <div className="col-12 my-2">
+                            <Desc value={'Amount'} variant={'dark'} />
+                            <input type="text" placeholder="Amount you will burn" className="form-control" id="BurnTokenAmount" name="BurnTokenAmount" />
+                        </div>
+                    </div>
+                </div>
+                <div className="modal-footer">
+                    <button type="button" className="btn btn-secondary" data-bs-dismiss="modal">Close</button>
```

Comment:
can we have a generic button component please. We are still using bootstrap and not styled-components.